printer_mods/slitte/ssd_bracket/readme.md

```diff
@@ -0,0 +1,21 @@
+# ssd_bracket
+bracket to mount a ssd-drive to the din rail under a voron printer
+
+This bracket is designed to mount a ssd drive in the electornics bay of a voron printer. 
+
+you need this additional part: https://github.com/VoronDesign/Voron-2/blob/Voron2.4/STLs/Electronics_Bay/pcb_din_clip_x3.stl
```

Comment:
same thing, please link the specific commit in case these change.

Reply:
linked the file to a specific commit